storage service endpoint

OrGur1987 opened the pull request:
see issue https://github.com/ManageIQ/manageiq-ui-classic/issues/8616

agrare commented:
@OrGur1987 can you add a screenshot of what this looks like?

agrare commented:
I added it to the description, having it in the PR description makes it simpler for the maintainers to build sprint-review slides

OrGur1987 commented:
> I added it to the description, having it in the PR description makes it simpler for the maintainers to build sprint-review slides

Ok, added the 2nd relevant screenshot.
Will add screenshots to PRs in issues in the future
